api/helpers/gamestate/convert-str-to-card.js

```diff
@@ -0,0 +1,127 @@
+
+//Suit: ‘C’ (Clubs), ‘D’ (Diamonds), ‘H’ (Hearts), ‘S’ (Spades)
+const suitMap = {
+  'C': 0,
+  'D': 1,
+  'H': 2,
+  'S': 3
+};
+
+//Rank: ‘A’ (Ace), ‘2’, ‘3’, ‘4’, ‘5’, ‘6’, ‘7’, ‘8’, ‘9’, ‘T’ (Ten), ‘J’ (Jack), ‘Q’ (Queen), ‘K’ (King)
+const rankMap = {
+  'A': 1,
+  'T': 10,
+  'J': 11,
+  'Q': 12,
+  'K': 13
+};
```

Comment:
Let's just add the 'unchanged' ranks to the map i.e. 2-9 like `'2': 2` so that we can simplify type conversion below